[2025-02-08T09:35:26.354+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T09:35:26.531+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Incident_Report_Generation.extraer_logs manual__2025-02-08T09:35:00.070958+00:00 [queued]>
[2025-02-08T09:35:26.543+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Incident_Report_Generation.extraer_logs manual__2025-02-08T09:35:00.070958+00:00 [queued]>
[2025-02-08T09:35:26.544+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-08T09:35:26.563+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extraer_logs> on 2025-02-08 09:35:00.070958+00:00
[2025-02-08T09:35:26.582+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=6603) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-08T09:35:26.583+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'Incident_Report_Generation', 'extraer_logs', 'manual__2025-02-08T09:35:00.070958+00:00', '--job-id', '309', '--raw', '--subdir', 'DAGS_FOLDER/Incident_Report_Generation.py', '--cfg-path', '/tmp/tmpepo2b1xq']
[2025-02-08T09:35:26.584+0000] {standard_task_runner.py:72} INFO - Started process 6612 to run task
[2025-02-08T09:35:26.584+0000] {standard_task_runner.py:105} INFO - Job 309: Subtask extraer_logs
[2025-02-08T09:35:26.663+0000] {task_command.py:467} INFO - Running <TaskInstance: Incident_Report_Generation.extraer_logs manual__2025-02-08T09:35:00.070958+00:00 [running]> on host 0b0ecc2e9e80
[2025-02-08T09:35:26.811+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Incident_Report_Generation' AIRFLOW_CTX_TASK_ID='extraer_logs' AIRFLOW_CTX_EXECUTION_DATE='2025-02-08T09:35:00.070958+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-08T09:35:00.070958+00:00'
[2025-02-08T09:35:26.817+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-08T09:35:26.818+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-08T09:35:26.818+0000] {logging_mixin.py:190} INFO - Current task name:extraer_logs state:running start_date:2025-02-08 09:35:26.531913+00:00
[2025-02-08T09:35:26.819+0000] {logging_mixin.py:190} INFO - Dag name:Incident_Report_Generation and current dag run status:running
[2025-02-08T09:35:26.820+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T09:35:26.847+0000] {base.py:84} INFO - Retrieving connection 'mysql_default'
[2025-02-08T09:35:27.370+0000] {logging_mixin.py:190} INFO - 3086
[2025-02-08T09:35:27.618+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T09:35:27.638+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T09:35:27.639+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=Incident_Report_Generation, task_id=extraer_logs, run_id=manual__2025-02-08T09:35:00.070958+00:00, execution_date=20250208T093500, start_date=20250208T093526, end_date=20250208T093527
[2025-02-08T09:35:27.670+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-08T09:35:27.673+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-08T09:35:27.673+0000] {logging_mixin.py:190} INFO - Dag name:Incident_Report_Generation queued_at:2025-02-08 09:35:00.431545+00:00
[2025-02-08T09:35:27.674+0000] {logging_mixin.py:190} INFO - Task hostname:0b0ecc2e9e80 operator:PythonOperator
[2025-02-08T09:35:27.704+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T09:35:27.768+0000] {taskinstance.py:3895} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2025-02-08T09:35:27.773+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
